src/pysdmx/io/xml/sdmx21/reader/metadata_read.py

```diff
@@ -64,10 +72,17 @@
     Facets,
 )
 from pysdmx.model.__base import Agency, Annotation, Contact, Item, ItemScheme
+from pysdmx.model.dataflow import Dataflow, DataStructureDefinition
 from pysdmx.model.message import CONCEPTS, ORGS
-from pysdmx.util import find_by_urn
-
-SCHEMES_CLASSES = {CL: Codelist, AGENCIES: ItemScheme, CS: ConceptScheme}
+from pysdmx.util import find_by_urn, parse_urn
+
+SCHEMES_CLASSES = {
```

Comment:
Should this be renamed to `SCHEMA_CLASSES` to avoid mixing with Item Schemes?

Reply:
We will rename it to MAPPING_(something here) as it is just a mapping from the key to the actual class

Reply:
Renamed